src/ads1x15/ads1015.cxx

```diff
@@ -22,6 +22,8 @@
  * WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
  */
 #include "ads1015.hpp"
+#include "mraa.hpp"
+#include "mraa/i2c.hpp"
```

Comment:
I see a missing include for usleep and sleep which must come from mraa.  Most likely some of these should also include unistd.h